src/view/app/Components/Layout/Query/QueryFormHead.tsx

```diff
@@ -21,6 +29,7 @@ interface QueryFormHeadProps
     onButtonClick?: MouseEventHandler<HTMLButtonElement>;
     formatButtonOnClick: MouseEventHandler<HTMLButtonElement>;
     isFormatted: boolean;
+    setIsFormatted: (value: boolean) => void;
```

Comment:
When you hover on the `useState` function, you can get the type and copy it here. Example:
![image](https://github.com/BalticAmadeus/ProBro/assets/44312404/2bebd16c-525c-413c-b01e-3cd4f8f74ebc)